src/object.c

```diff
@@ -1202,48 +1202,51 @@ size_t streamRadixTreeMemoryUsage(rax *rax) {
     return size;
 }
 
-/* Returns the size in bytes consumed by the key's value in RAM.
- * Note that the returned value is just an approximation, especially in the
+/* Returns the size in bytes consumed by the object header, key and value in RAM.
+ * Note that this is just an approximation, especially in the
```

Comment:
The old comment seems fine too.

Reply:
Old comment talks only about the value

Reply:
no, i mean `the returned value` is okay.